lib/chef/provider/cookbook_file.rb

```diff
@@ -36,15 +36,15 @@ def initialize(new_resource, run_context)
       end
 
       def load_current_resource
-        @current_resource = Chef::Resource::CookbookFile.new(@new_resource.name)
+        @current_resource = Chef::Resource::CookbookFile.new(new_resource.name)
```

Comment:
It saddens me a little that we don't have some sugar around this pattern, given how often we repeat it.